<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="View">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_export">
    <attribute name="title">
     <string>Export</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton_pack0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to the pack0 folder</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_custom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to an other folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_selectDir">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_exportFolder">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>60</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Diffuse map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mtNormalsMap">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Normals map</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_moveTextures">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy textures</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mtSpecularMap">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Specular map</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_format">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>.jpg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>.png</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>.tga</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>.pcx</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>.ppm</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_convertTextures">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Convert textures (DDS format by default)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tw3">
    <attribute name="title">
     <string>TW3 settings</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_TW3_selectTexFolder">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TW3_texFolder">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_TW3_texFolder">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Textures folder :</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_TW3_loadSkel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Load skeleton (experimental - disable it if you don't need it)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_TW3_loadBestLOD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Load only the best LOD</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_view">
    <attribute name="title">
     <string>View</string>
    </attribute>
    <widget class="QLabel" name="label_camera">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cam√©ra</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_cameraSpeed">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_cameraRotSpeed">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="button_colorSelector">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_colorSelector">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Background color</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_movementSpeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Movement Speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rotSpeed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation speed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_debug">
    <attribute name="title">
     <string>Debug</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_debug">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable &quot;debug.log&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_debugLog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>If you have a problem to load a w2ent/w2mesh file, you can enable this option to generate a log file. This file can help me to solve your problem, so in case of problem, you can send me this file (via the forum or by mail at jeanlouis[dot]boudrand[at]hotmail[dot]fr.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
